static/app/components/events/eventTags/eventTagsTree.tsx

```diff
@@ -110,21 +136,105 @@ function TagTreeRow({
+              projectSlug={projectSlug}
+              releaseVersion={content.value}
+              showUnderline
+              underlineColor="linkUnderline"
+            >
+              <Version version={content.value} truncate />
+            </VersionHoverCard>
           ) : (
             content.value
           )}
         </TreeValue>
+        <TreeValueDropdown
+          preventOverflowOptions={{padding: 4}}
+          className={!isVisible ? 'invisible' : ''}
+          position="bottom-start"
+          dropdownOverlayProps={{onOpenChange: isOpen => setIsVisible(isOpen)}}
+          triggerProps={{
+            'aria-label': t('Tag Actions Menu'),
+            icon: <IconEllipsis />,
+            showChevron: false,
+            size: 'xs',
+            className: 'tag-button',
+          }}
+          items={[
+            {
+              key: 'view-events',
+              label: t('View other events with this tag value'),
+              hidden: !event.groupID,
+              onAction: () => {
+                navigateTo(
+                  `/organizations/${organization.slug}/issues/${event.groupID}/events/${searchQuery}`,
+                  router
+                );
+              },
+            },
+            {
+              key: 'view-issues',
+              label: t('View issues with this tag value'),
+              onAction: () => {
+                navigateTo(
+                  `/organizations/${organization.slug}/issues/${searchQuery}`,
+                  router
+                );
+              },
+            },
+            {
+              key: 'release',
+              label: t('View this release'),
+              hidden: originalTag.key !== 'release',
+              onAction: () => {
+                navigateTo(
+                  `/organizations/${organization.slug}/releases/${encodeURIComponent(
+                    content.value
+                  )}/`,
+                  router
+                );
+              },
+            },
+            {
+              key: 'transaction',
+              label: t('View this transaction'),
+              hidden: originalTag.key !== 'transaction',
+              onAction: () => {
+                const transactionQuery = qs.stringify({
+                  project: event.projectID,
+                  transaction: content.value,
+                  referrer,
+                });
+                navigateTo(
+                  `/organizations/${organization.slug}/performance/summary/?${transactionQuery}`,
+                  router
+                );
+              },
+            },
+            {
+              key: 'replay',
+              label: t('View this replay'),
+              hidden: originalTag.key !== 'replay_id',
+              onAction: () => {
+                const replayQuery = qs.stringify({referrer});
+                navigateTo(
+                  `/replays/${encodeURIComponent(content.value)}/?${replayQuery}`,
+                  router
```

Comment:
needs /org/org-slug i think

Reply:
self hosting requires the `/organization/{slug}` since the customerDomain feature is disabled